### PR TITLE
[libyaml] update to 0.2.5

### PR DESCRIPTION
**Pull request by nephros**
Follow-up on https://github.com/sailfishos/python3-yaml/pull/1

Now that we depend on libyaml in python3-yaml, the lib could probably benefit from a bump as well. The current version (in SFOS) is 0.1.4 from 2014 (plus some security fixes).

Is there any interest in going forward with that?

(Packages available for testing at [OBS](https://build.merproject.org/project/show/home:nephros:sailfishos:python)).

### CODE REVIEW

**Comment by pvuorela** [comment]
Wouldn't see reasons not to update this.

What's the draft status here? Seems to be doing what it's supposed to already.

For one thing I could rearrange and squash the commits, now there are some changes back and forth. Instead first the commit for removing old sources and then all the other things in another commit.

**Comment by nephros** [comment]
Ok, squashed tree coming up!

**Review by pvuorela — APPROVED** [review]
Looking good. Thanks!

File: rpm/libyaml.spec [hunk 1]
```diff
@@ -45,13 +45,11 @@ make check
 
 %files
 %defattr(-,root,root,-)
-%doc LICENSE README
+%doc License ReadMe.md
```
Comment (saukko) [inline rpm/libyaml.spec:48]: The License file should be with %license macro. Maybe another PR about that?

**Pull request merged**